wiki/Tournaments/o!BAM/1/en.md

```diff
@@ -0,0 +1,576 @@
+| **5Digitbutn00b** | ![][flag_MY] **[lil scarce](https://osu.ppy.sh/users/11299087)**, ![][flag_MY] [D4rK4C1D](https://osu.ppy.sh/users/19727967) |
+| **Brown Switch Gaming** | ![][flag_MY] **[LITFAM123](https://osu.ppy.sh/users/13584478)**, ![][flag_MY] [Chiew](https://osu.ppy.sh/users/11378724) |
+| **IDoubleTapJumpMaps** | ![][flag_MY] **[GodKnows](https://osu.ppy.sh/users/11058193)**, ![][flag_MY] [IHateAutoNicks](https://osu.ppy.sh/users/10758397) |
+| **Ascendants** | ![][flag_MY] **[Inugami Korone](https://osu.ppy.sh/users/4474918)**, ![][flag_MY] [Mumei Nanashi](https://osu.ppy.sh/users/5414124/osu) |
+| **ur mom** | ![][flag_MY] **[Houshou Marine](https://osu.ppy.sh/users/6402211)**, ![][flag_MY] [Banner](https://osu.ppy.sh/users/14290988) |
+| **"oh no"** | ![][flag_MY] **[RushiaFanGirl](https://osu.ppy.sh/users/11918602)**, ![][flag_MY] [Illuxz](https://osu.ppy.sh/users/11916798) |
+| **Snowflakes fluttering north wind howling** | ![][flag_MY] **[JKLiew02](https://osu.ppy.sh/users/17434745)**, ![][flag_MY] [Iyouka](https://osu.ppy.sh/users/7138499) |
+| **Chicken Sandwich** | ![][flag_MY] **[fernsfurn](https://osu.ppy.sh/users/14831389)**, ![][flag_MY] [Fuurin_Desu](https://osu.ppy.sh/users/18173411) |
+| **LOLI ANNIHILATOR** | ![][flag_MY] **[xanotic](https://osu.ppy.sh/users/18097295)**, ![][flag_MY] [[MY]ISPROGILER](https://osu.ppy.sh/users/10911621) |
+| **poggers9000** | ![][flag_MY] **[Hinnoa](https://osu.ppy.sh/users/11001723)**, ![][flag_MY] [Havvik](https://osu.ppy.sh/users/9868585) |
+| **Sheep** | ![][flag_MY] **[FaithInOsu](https://osu.ppy.sh/users/10178187)**, ![][flag_MY] [Vexine](https://osu.ppy.sh/users/4042706) |
+| **i just cooked cheese** | ![][flag_MY] **[RyzenPP](https://osu.ppy.sh/users/16391241)**, ![][flag_MY] [NAVolkan](https://osu.ppy.sh/users/14981515) |
+| **Chicken Mutton** | ![][flag_MY] **[Inequality](https://osu.ppy.sh/users/8455034)**, ![][flag_MY] [Hiqal](https://osu.ppy.sh/users/13892584) |
+| **Penguin Enjoyers** | ![][flag_MY] **[wantoon](https://osu.ppy.sh/users/16042959)**, ![][flag_MY] [CXC1](https://osu.ppy.sh/users/12004902) |
+| **e** | ![][flag_MY] **[Eximious](https://osu.ppy.sh/users/19872191)**, ![][flag_MY] [_GRiD](https://osu.ppy.sh/users/20743023) |
+| **pen el grande** | ![][flag_MY] **[Zervail](https://osu.ppy.sh/users/13047340)**, ![][flag_MY] [DARKWEBXD](https://osu.ppy.sh/users/13060849) |
+| **amriourmommy** | ![][flag_MY] **[\[Fubuki\]](https://osu.ppy.sh/users/14411486)**, ![][flag_MY] [GideonTan](https://osu.ppy.sh/users/17426621) |
+| **Monkeybsns** | ![][flag_MY] **[ClawViper](https://osu.ppy.sh/users/2681361)**, ![][flag_MY] [Jeffrey](https://osu.ppy.sh/users/4777134) |
+| **aidanyunusfanclub** | ![][flag_MY] **[pootisman1547](https://osu.ppy.sh/users/6611419)**, ![][flag_MY] [#RESTRICTED](https://osu.ppy.sh/users/18897374) |
```

Comment:
hmm, is there any reason why the user here is censored? iirc tournament wiki pages are meant as a form of "documentation" of sorts so players who participate in the tournament but has since been restricted should still be included as is in the wiki

(see : the wiki pages of the earlier owc tournaments (e.g. [owc #1](http://osu.ppy.sh/wiki/en/Tournaments/OWC/1)) for reference)